manager/state/raft/raft_test.go

```diff
@@ -488,6 +493,7 @@ func TestRaftForceNewCluster(t *testing.T) {
 	t.Parallel()
 
 	nodes, clockSource := raftutils.NewRaftCluster(t, tc)
+	defer raftutils.TeardownCluster(t, nodes)
```

Comment:
I think this is already being cleaned up properly. See `raftutils.TeardownCluster(t, toClean)` and `defer raftutils.TeardownCluster(t, newCluster)`.